python/whylogs/api/whylabs/session_manager.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import uuid
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional, Union
+
+from whylabs_client import ApiException
+from whylabs_client.api.sessions_api import (
+    CreateSessionRequest,
+    CreateSessionResponse,
+    SessionsApi,
+)
+from whylabs_client.api_client import ApiClient, Configuration
+
+from whylogs.api.whylabs.auth_file import get_auth_file_path
+from whylogs.api.whylabs.notebook_check import is_notebook
+from whylogs.api.whylabs.variables import Variables
+
+DEFAULT_WHYLABS_HOST = "https://api.whylabsapp.com"
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class GuestSession:
+    session_id: str
+
+
+@dataclass
+class ApiKeySession:
+    org_id: str
+    api_key: str
+
+
+def _get_default_authentication_path():
+    default_path = os.getenv("WHYLOGS_CONFIG_PATH") or f"{Path.home()}/.whylabs/auth.ini"
+    auth_path = get_auth_file_path(auth_path=Path(default_path))
+    return auth_path
+
+
+def _create_session_id(user_id: str) -> str:
+    config = Configuration()
+    config.host = os.getenv("WHYLABS_API_ENDPOINT") or DEFAULT_WHYLABS_HOST
+
+    try:
+        client = ApiClient(config)
+        api = SessionsApi(client)
+        response: CreateSessionResponse = api.create_session(CreateSessionRequest(user_id))
+
+        logger.debug(f"Created session {response.id}")
+        return response.id
+    except ApiException as e:
+        logger.error(e)
+        raise e
+
+
+def _get_logged_session(auth_path: Optional[Path] = None) -> ApiKeySession:
+    api_key = os.getenv("WHYLABS_API_KEY")
+    org_id = os.getenv("ORG_ID")
+
+    if api_key is None or org_id is None:
+        _auth_path = _get_default_authentication_path()
+        api_key = api_key or Variables.get_variable_from_config_file(auth_path=auth_path, key="api_key")
+        org_id = org_id or Variables.get_variable_from_config_file(auth_path=auth_path, key="org_id")
+
+        if is_notebook():
+            if api_key is None:
+                api_key = Variables.get_variable_from_getpass(variable_name="api_key")
+                Variables.set_variable_to_config_file(key="api_key", value=api_key, auth_path=_auth_path)
+            if org_id is None:
```

Comment:
It'll be weird to use an API key from a config file and an org ID from an environment variable. It should be all or nothing.

It's worth printing out where you're pulling the variables from as well, if not in the debug code. It's important for debugging